lambdas/data-migration2/tests/test-granulesAndFiles.js

```diff
@@ -116,11 +121,13 @@ test.beforeEach(async (t) => {
   t.context.testExecution = testExecution;
   t.context.executionCumulusId = executionResponse[0];
 
-  t.context.testGranule = generateTestGranule(testCollection, testExecution.arn);
+  t.context.testGranule = generateTestGranule(testCollection, executionUrl);
 });
 
 test.afterEach.always(async (t) => {
+  // TODO: we should be just removing the records via `delete` methods
```

Comment:
Agree,  now that we're using the DB creation, there's no value in full wipes, and all these are .serial because of it.    We should address this in another PR.